### PR TITLE
[TASK] Change default caching framework names (#1253)

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves: #1253

### CODE REVIEW

**Comment by tomasnorre** [comment]
I was a little eager to move on, so I continued with this one. 
I expect the feedback from #1059 will apply to this one too, but will see. :) 

This one isn't working yet though. I have commented on the code where I have problems. But think it makes more sense to wait for the feedback.

File: src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        return [MethodCall::class];
+    }
+
+    /**
+     * @param MethodCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if (! $this->isMethodStaticCallOrClassMethodObjectType($node, CacheManager::class)) {
```
Comment (tomasnorre) [inline src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php:32]: This check isn't working like I expected, always hit this one, If I remove this check and the 
```php 
if (! $this->isName($node->name, 'getCache')) {
    return null;
}
``` 

Then I can see that the change of the argument is working as expected. So not that far from something working. But there might be a simpler rule for this one too as mentioned in the comment on the PR itself.

Comment (sabbelasichon) [reply src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php:32]: @tomasnorre The things is this line here use TYPO3\CMS\Core\Core\CacheManager; Remove one of the core subprefix and it should work

File: src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v10\v0;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use TYPO3\CMS\Core\Core\CacheManager;
```
Comment (sabbelasichon) [inline src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php:12]: This is wrong. Change Core with Cache and it should work

Comment (tomasnorre) [reply src/Rector/v10/v0/ChangeDefaultCachingFrameworkNamesRector.php]: I overlooked that.

**Comment by sabbelasichon** [comment]
Very cool. One last thing: Could you please add the rector to the appropriate config file too?

**Comment by tomasnorre** [comment]
>Very cool. One last thing: Could you please add the rector to the appropriate config file too? 

Done, but that I didn't do for the first rector that you have already merged.

**Pull request merged**